example/Gemfile.lock

```diff
@@ -5,26 +5,23 @@ GEM
       base64
```

Comment:
Do we need to make these changes in Gemfile.lock?

Reply:
They happen automatically when I run bundle install so 🤷